lib/components/SideNote/index.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  ComponentProps,
+  MouseEventHandler,
+  PropsWithChildren,
+  ReactElement,
+} from "react";
+import clsx from "clsx";
+import { ChevronRightIcon } from "./ChevronRightIcon";
+import { Text } from "../Text";
+import "./SideNote.scss";
+
+type SideNoteProps = Omit<ComponentProps<"div">, "className"> & {
+  title?: string | ReactElement;
+  titleSize?: ComponentProps<typeof Text>["size"];
+  className?: HTMLDivElement["className"];
+  actionClick?: MouseEventHandler<HTMLButtonElement>;
+  actionClassName?: HTMLDivElement["className"];
+  actionLabel?: string | ReactElement;
```

Comment:
```suggestion
  actionLabel?: string | React.JSX.Element;
```
As I checked the translation library , `React.JSX.Element` would be the one we want in case the consumer wants to use `<Localize/>`. I've done the same thing for VerticalTab as well